hatch.toml

```diff
@@ -183,6 +183,7 @@ CMAKE_BUILD_PARALLEL_LEVEL = "12"
 ## ASM Django
 
 [envs.appsec_threats_django]
+python = ["3.7", "3.9", "3.10"]
```

Comment:
Does this mean these don't pass with >3.10 ?

Reply:
I think this was added unintentionally